### ISSUE TITLE
if studyDate is absent in curator all saves will fail

### ISSUE THREAD

**Issue by mtholder** [OpenTreeOfLife/opentree#477]
I'm not sure if we should call this a peyotl bug (being too picky) or curator issue. I created a bogus empty study and saved it:
 https://github.com/OpenTreeOfLife/phylesystem-0/commit/0ca8cdf09e63dedf326d137dd5a3ffe0c51a305a

Then when I added a tree and hit save, the save was rejected because the study had no year. The curator app shows a red warning at the bottom, but it is somewhat easy to miss. 

Perhaps we should figure out the minimal set of info, and hassle people until they fill that in so that they won't get confused about what content is being rejected.

Maybe the study year should fall into the warn-but-do-not-reject category of deficiency.

**Comment by jimallman** [comment]
Yes, it seems the study year is the **only** information required in a new study, which is odd. I can add client-side validation for this and other fields, but of course this won't affect someone using the phylesystem API directly. 

I agree that we should settle on a minimal set of required fields. Maybe this means none at all, if we want to allow many use cases where the curator has only partial data.

For now it might be great to relax the validator.
